src/SettingsCart.tsx

```diff
@@ -84,6 +91,7 @@ export  const SettingsCart: React.FC<SettingsCartParams> = (props) => {
             )}
             <div className="epform__error">
               {errors.name ? errors.name : null}
+              {values.name.length > 250 ? nameError : null}
```

Comment:
I know we removed the client side check here last time. Did we confirm with UX and QA that we're adding it back?

Reply:
Yes, We want to show the error, so user can edit the name and try again